src/store/index.js

```diff
@@ -10,12 +10,23 @@ Vue.use(Vuex)
 export default new Vuex.Store({
     state: {
         token: null,
+        isLogged: false,
+    },
+    mutations: {
+        loginUser (state) {
+            state.isLogged = true
+        },
+        logoutUser (state) {
+            state.isLogged = false
+        }
```

Comment:
as it's mutations for isLogged, you could create simple one setter with parameter

Reply:
another thing is, your team decide to use modules, so keep the convention and create module named 'auth' or smth like that